### ISSUE TITLE
PHPDoc comment with misplaced space - comment doesn't terminate

### ISSUE THREAD

**Issue by roblourens** [atom/language-php#256]
https://github.com/atom/language-php/pull/250 fixed the same issue for malformed types in phpdoc. Here's another case.

Have a phpdoc comment with a space after `@param`, like here:

```php
/**
* @param (int) number that will be printed
*
*/
function foo($num) {
    echo $num;
}


function bar($num) {
    echo "Why is this commented out";
}
```

See that the rest of the document has `meta.other.type.phpdoc.php`:
![](https://user-images.githubusercontent.com/368981/29285607-d9d2be38-80e3-11e7-97aa-58395aa916b9.png)

https://github.com/Microsoft/vscode/issues/32500

**Comment by 50Wliu** [comment]
Cannot reproduce:
![php-incorrect-phpdoc](https://user-images.githubusercontent.com/2766036/29292962-35334d28-8117-11e7-97cc-2e60748e18f5.png)

**Comment by roblourens** [comment]
Is that with the latest grammar? I can also repro in Atom with the latest grammar loaded, quite easily:

![image](https://user-images.githubusercontent.com/323878/29293957-ec97a4d4-8101-11e7-88df-23980e2bf488.png)

**Comment by Ingramz** [comment]
@roblourens try one of the more recent builds, such as [this one](https://ci.appveyor.com/project/Atom/atom/build/7653/job/xs7eov61u95q0t81/artifacts).

**Comment by 50Wliu** [comment]
Specifically, I believe that this was fixed by #250, which should be included in the Atom 1.20.0 betas.

**Comment by roblourens** [comment]
You're right - I wasn't using the build of vscode that I thought I was, and apparently I messed up running Atom in dev mode.
